src/main/java/org/apache/cassandra/sidecar/metrics/MetricsPublisher.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.metrics;
+
+/**
+ * Interface used for publishing metrics across endpoints in Sidecar.
+ */
+public interface MetricsPublisher
+{
+    void captureCount(Metric metric);
+
+    void captureAggregation(Metric metric, long value);
```

Comment:
`captureAggregation` is invoked in both `SSTableUploadHandler` and `FileStreamer`. How does an actual publisher distinguish the metrics?